### ISSUE TITLE
Problem downloading and running a script that will install HNN

### ISSUE THREAD

**Issue by andy-nord** [jonescompneurolab/hnn#312]
I am new to all this and currently making first steps. When I am trying to download a script to install HNN and type curl --remote....ect I receive messages curl: (6) Could not resolve host: bash and curl: (6) Could not resolve host: hnn-ubuntu.sh. Why does this error appear?

**Comment by jasmainak** [comment]
@andy-nord I don't see a reason why that should fail. Most likely you had a temporary network outage. If you try again, it should work

FYI, the HNN-GUI is not actively maintained at the moment. What you might want to do instead is install `hnn-core`:

https://jonescompneurolab.github.io/hnn-core/dev/index.html

and follow the tutorials along with the GUI tutorials. We can of course help you with the GUI installation if you insist going that route

**Comment by andy-nord** [comment]
Yes, it worked this time. Thank you! I downloaded Python and tried to get hnn-core in according to instructions https://jonescompneurolab.github.io/hnn-core/dev/index.html
but it didn't work.

**Comment by andy-nord** [comment]
Update. I just noticed that instead of Ubuntu I used Windows Command Prompt and it downloaded HNN together with NEURON and numpy and installed these. Now when I am trying to do the same in Ubuntu prompt it doesn't work again. Next step from tutorial Start HNN from https://jonescompneurolab.github.io/hnn/installer/windows/ doesn't work either. How to start with HNN from scratch?

**Comment by jasmainak** [comment]
Sorry, what do you mean that it didn't work? What did you try and what message did you see?

**Comment by andy-nord** [comment]
I downloaded Ubuntu 18.04 LTS and turned on WSL in Windows features. I also downloaded VcXsrv and went through all steps related to VcXsrv described here https://jonescompneurolab.github.io/hnn/installer/windows/wsl.html
Now when I am trying to run next step curl --remote-name https://raw.githubusercontent.com/jonescompneurolab/hnn/master/installer/ubuntu/hnn-ubuntu.sh
it shows only several 0s
when I am trying to run pip install hnn_core in Python I receive this error
ERROR: Command errored out with exit status 1:
   command: 'C:\Users\user\anaconda3\python.exe' 'C:\Users\user\anaconda3\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\user\AppData\Local\Temp\pip-build-env-la3wuh0f\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'setuptools>=40.8.0' 'NEURON >=7.7'
       cwd: None
  Complete output (4 lines):
  Collecting setuptools>=40.8.0
    Using cached setuptools-57.4.0-py3-none-any.whl (819 kB)
  ERROR: Could not find a version that satisfies the requirement NEURON>=7.7
  ERROR: No matching distribution found for NEURON>=7.7

**Comment by jasmainak** [comment]
NEURON does not have a pip distribution on Windows. I'm not sure if the WSL route can fix the problem, I have never used it. @ntolley any idea? 

Alternatively you could download NEURON from here: https://www.neuron.yale.edu/neuron/download as [we recommend](https://jonescompneurolab.github.io/hnn-core/dev/index.html) on hnn-core documentation. On Ubuntu, you should not have any problem with `pip install hnn_core`. I think some of our developers are using it without any problems

**Comment by ntolley** [comment]
@andy-nord just to confirm is the above error message in Windows command prompt or WSL (Ubuntu)?

Also appologies we are still behind on getting the native windows install in a stable state, the best route at the moment is WSL.

**Comment by andy-nord** [comment]
First error I received in Windows prompt when I used it by mistake instead of Ubuntu. I could download HNN by this way but could not execute second step and launch HNN. When I tried to use Ubuntu it didn't download HNN at all. When Ubuntu didn't work I followed advice from @jasmainak to install hnn-core from here https://jonescompneurolab.github.io/hnn-core/dev/index.html - Last error which I provided in the previous message was in Python. 
Maybe you have any plans to create a video tutorial for beginners? This would help a lot in making things easier. HNN is fascinating tool but right now I am a little bit confused by multiple versions of HNN/ways to get it and numerous pre-requisites.

**Comment by ntolley** [comment]
Really appreciate the feedback, a video is definitely a great idea. This week I will look to update the documentation for hnn-core to better clarify the installation.

I'll need to dig into the error message for HNN-GUI on windows, but in the mean time I think the best path is installing hnn-core on Ubuntu/WSL. If you run into issues on that route, I am happy to set up a meeting to troubleshoot.

**Comment by jasmainak** [comment]
Agree that looking into the Windows installation issue before our release is a great idea!

**Comment by andy-nord** [comment]
I would definitely appreciate any help/clarifications. I am afraid I will not be able to resolve this alone.

**Comment by ntolley** [comment]
Please feel free to email me at nicholas_tolley@brown.edu and we can set up a time to chat!
